test/e2e/functional-header.test.js

```diff
@@ -0,0 +1,44 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlFunctionalHeaderPage = require('./pages/vl-functional-header.page');
+
+describe('vl-functional-header', async () => {
+    const vlFunctionalHeaderPage = new VlFunctionalHeaderPage(driver);
+
+    before(() => {
+        return vlFunctionalHeaderPage.load();
+    });
+
+    it('als gebruiker kan ik de titel en sub titel van de functionele header zien', async () => {
+        const functionalHeader = await vlFunctionalHeaderPage.getFunctionalHeader();
+        const title = await functionalHeader.getTitle();
+        const subTitle = await functionalHeader.getSubTitle();
+        await assert.eventually.equal(title.getText(), 'SCHOOL- EN STUDIETOELAGEN');
+        await assert.eventually.equal(subTitle.getText(), 'Voor lager, middelbaar en hoger onderwijs');
+    });
+
+    it('als gebruiker kan ik de titel en sub titel van de functionele header zien als ze als slot gedefinieerd werden', async () => {
+        const functionalHeader = await vlFunctionalHeaderPage.getFunctionalHeaderSlots();
+        const title = await functionalHeader.getTitle();
+        const subTitle = await functionalHeader.getSubTitle();
+        await assert.eventually.equal(title.getText(), 'SCHOOL- EN STUDIETOELAGEN');
+        await assert.eventually.equal(subTitle.getText(), 'Voor lager, middelbaar en hoger onderwijs');
```

Comment:
moet je dan niet eerder een element in het slot steken en zien dat je dat nog kunt opvragen?

Reply:
De tekst zit in een element dat in een slot zit, dus is gedekt?

Reply:
mja ... ik zou voor het gebruik van slots aan te tonen altijd nog een element in het slot steken maar puur theoretisch heb je gelijk zeker

Reply:
Ik zal de slot functies alsnog voorzien want momenteel faalt de test in Firefox.